practices/language.md

```diff
@@ -117,8 +120,14 @@ Follow Plain English guidelines to help safeguard the accessibility of your writ
   - Be as precise as you can usefully be - consider linking to high quality external resources where relevant.
 
 
+## Frontend or front-end
+
+* Use `front-end` when you need an adjective (e.g. "the front-end team", "front-end principles").
+* Use `frontend` when you need a noun (e.g. "the Shunter frontend"), when you refer to a repository, or in your code.
```

Comment:
The example is a bit esoteric...  Perhaps "the application's frontend" would be a better one?